src/main/java/org/mockito/ArgumentCaptor.java

```diff
@@ -100,12 +84,41 @@ private ArgumentCaptor(Class<? extends T> clazz) {
      * @return null or default values
      */
     public T capture() {
-        Mockito.argThat(capturingMatcher);
+        return captureIf(Any.ANY);
```

Comment:
@ChristianSchwarz I implemented the last suggestion you had. Please take a look and let me know if you don't understand. Thanks.